src/components/NavSection.vue

```diff
@@ -176,7 +176,7 @@ li {
 }
 
 /* 48em = 768px */
-@media (min-width: 49em) {
+@media (min-width: 55em) {
```

Comment:
Thanks for spotting this. But on my testing here, this doesn't need to be such a big change. `52em` here fixed the issue. Can you change it please?